libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationTelemetry.java

```diff
@@ -88,7 +89,10 @@ public void onDeparture(Location location, RouteProgress routeProgress) {
     if (metricProgress == null) {
       metricProgress = new MetricsRouteProgress(routeProgress);
     }
-    NavigationMetricsWrapper.departEvent(navigationSessionState, metricProgress, location);
+    if (!hasDeparted) {
```

Comment:
In a second pass, we should do this check up a level, so that `onDeparture()` isn't called multiple times.

Reply:
@ericrwolfe this is done by removing the listener in #540